draft-looker-oauth-attested-key-based-client-authentication.md

```diff
@@ -55,20 +55,22 @@ informative:
 
 --- abstract
 
-This specification defines a new method of client authentication for OAuth2 {{RFC6749}} by extending the approach defined in {{RFC7521}}. This new method enables client deployments that are traditionally viewed as public clients to be able to authenticate with the authorization server through an attested key based authentication scheme.
+This specification defines a new method of client authentication for OAuth 2.0 {{RFC6749}} by extending the approach defined in {{RFC7521}}. This new method enables client deployments that are traditionally viewed as public clients to be able to authenticate with the authorization server through an attestation based authentication scheme.
```

Comment:
Just noting that given we now have OAuth 2.1 saying this is for OAuth 2.0 might give the wrong compatibility impression?

Reply:
@tlodderstedt originally made this change

Reply:
we don't have OAuth 2.1 yet, it's a draft. But I would be fine with just "OAuth" (with the existing reference to RFC 6749